src/lib/Repository/UserService.php

```diff
@@ -123,6 +126,7 @@ public function __construct(
         $this->passwordHashService = $passwordHashGenerator;
         $this->passwordValidator = $passwordValidator;
         $this->configResolver = $configResolver;
+        $this->logger = new NullLogger();
```

Comment:
You need `use LoggerAwareTrait;` + the following:
```suggestion
        $this->logger = $logger ?? new NullLogger();
```